src/ReadVTK.jl

```diff
@@ -909,11 +910,14 @@ function get_coordinates(vtk_file::VTKFile; x_string="x", y_string="y", z_string
       error("The file type of the VTK file must be 'RectilinearGrid' (current: $(vtk_file.file_type)).")
   end
 
-  coordinates = get_coordinate_data(vtk_file)
-  x = get_data(coordinates[x_string])
-  y = get_data(coordinates[y_string])
-  z = get_data(coordinates[z_string])
+  if  vtk_file.file_type == "RectilinearGrid"
+    coordinates = get_coordinate_data(vtk_file)
+    x = get_data(coordinates[x_string])
+    y = get_data(coordinates[y_string])
+    z = get_data(coordinates[z_string])
 
+  end
+  
```

Comment:
```suggestion
  coordinates = get_coordinate_data(vtk_file)
  x = get_data(coordinates[x_string])
  y = get_data(coordinates[y_string])
  z = get_data(coordinates[z_string])
```
It does not make sense to put this in an if condition, since it was already checked in lines 909ff that this mist be a file of type `RectilinearGrid`. Or am I misssing something?

Reply:
correct